tests/tests.rs

```diff
@@ -109,11 +109,23 @@ fn test_pause() {
     let (tx, rx) = mpsc::channel();
     thread::spawn(move || {
         // pause
-        tx.send(f()).unwrap();
+        {
+            f();
+            tx.send(())
+        }
+        .unwrap();
         // woken up by new order pause, and then pause again.
-        tx.send(f()).unwrap();
+        {
+            f();
+            tx.send(())
+        }
+        .unwrap();
         // woken up by remove, and then quit immediately.
-        tx.send(f()).unwrap();
+        {
```

Comment:
Why change this?

Reply:
make clippy happy

Reply:
What's the error? I can't reproduce it locally.

Reply:
cargo clippy --tests -- -D warnings

Reply:
How about
```
f();
tx.send(()).unwrap();
```
? So brackets are not necessary.

Reply:
Done